libs/libmilter/patches/101-pthreads-stack-size.patch

```diff
@@ -0,0 +1,34 @@
+Index: sendmail-8.15.2/libmilter/libmilter.h
```

Comment:
Done. Please review [the comment](https://github.com/openwrt/packages/pull/4369/commits/4bce6115cfbde375f50ff84c486445cf3dc07d8b#diff-9c8da989d26bb43cf9df88f9e9848d91).